README.md

```diff
@@ -1,575 +1,17 @@
 # Cloud Native Taiwan User Group
+Cloud Native Computing Taiwan User Group 希望整合台灣雲端相關社群如 OpenStack, Kubernetes, Ceph, SDN 等，並利用 Meetup 定期在各台灣地區舉辦技術與推廣分享活動。目的是藉此群策群力來使該社團快速成長，以對台灣在雲端開源平台的貢獻，另外我們更希望該社群持有 Cloud Native 概念，並將此推廣至台灣企業、學校、政府單位等。
 
-- [Events & slides](communities/CNTUG.md)
-- Location: 台中(Taichung)/台北(Taipei)/台灣(Taiwan)
-- Keywords: `cloud native`, `DevOps`, `open source`,`SDN`,`OpenStack`,`SDS`
+Cloud Native Computing Taiwan User Group is dedicated to combine community of Cloud Native related projects such as Kubernetes, OpenStack, Ceph, SDN. We held meetups regularly to share related technologies and promote them. We wish to grow this user group with the help from community members to increase contributions to Cloud Native Platforms in Taiwan. We also wish to promote the concept of "Cloud Native" to enterprise, academic institutes, and governments.
 
-###### `cloud native`, `DevOps`, `open source`,`SDN`,`OpenStack`,`SDS`
+- [Meetup](https://www.meetup.com/CloudNative-Taiwan/)
+- [Facebook](https://fb.cloudnative.tw)
+  - [SDNDS Taiwan](https://www.facebook.com/groups/sdnds.tw/) - Cooperative community.
+  - [Golang Taiwan](https://www.facebook.com/groups/269001993248363/) - Cooperative community.
```

Comment:
Add the facebook link of CNTUG ?
https://www.facebook.com/groups/cloudnative.tw/